modules/ROOT/partials/configuration/importword.adoc

```diff
@@ -0,0 +1,33 @@
+[[importword-service-url-option]]
+== `importword_service_url` option
+
+This option is used for setting the URL endpoint for the conversion service.
+
+[IMPORTANT]
+If this string value is **not configured** it will return a console.error:
+_The {pluginname} plugin requires the `importword_service_url` to be configured_. If the editor detects an `invalid_URL`, it will return a console.error: _The value provided in `importword_service_url` is not a valid URL_.
+
+*Type:* `+String+`
+
+=== Example: using `importword_service_url`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'code importword',
+  toolbar: 'importword code',
+  importword_service_url: '<replace-with-service-url>'
+});
+----
+
+=== Limitation(s) of the `importword_service_url` option
+
+The `importword_service_url` option has the following limitations.
+
+TODO//
+////
+Known limitations.
```

Comment:
@kemister85  We need add this to know issues /limitations 

Note from @hamza0867 

> Yes this is a “limitation” from Safari and Firefox browsers. These browsers do not allow opening a file picker programmatically without “true” user interaction. So calling the command from the console won’t work but if you call the command in the callback of a user interaction then it works as intended as you can see when you click on the importword toolbar button.
> 
> So this issue should just be documented as a limitation of the browsers. 

Reference: https://ephocks.atlassian.net/browse/TINY-10716